### DISCUSSION TITLE
Jetbrains cycling through suggestions doesn't work

### DISCUSSION THREAD

**Discussion by ta32** [community/community#6999] [Copilot Conversations] [answered]
When I open co-pilot window it shows multiple suggestions, however the key bindings to cycle through suggestions did not work.

I thought it was because of keyboard conflict, so I changed the key bindings and it still doesn't cycle through the suggestions. 

* **IntelliJ Rust plugin version:** 0.4.158.4203-212
* **Rust toolchain version:** 1.56.0 (09c42c458 2021-10-18) x86_64-pc-windows-msvc
* **IDE name and version:** CLion 2021.2.3 (CL-212.5457.51)
* **Operating system:** Windows 10 10.0
* **Macro expansion engine:** new
* **Name resolution engine:** new

![image](https://user-images.githubusercontent.com/26617779/139434912-72337ab3-df35-49a8-aaf4-fc9784c78f2c.png)

even with the key bindings changed it does not cycle
![image](https://user-images.githubusercontent.com/26617779/139435089-64befb67-d7d1-4c4c-9a25-50d36100508c.png)

**Comment by jansorg** [comment, marked as answer]
These shortcuts are actually for the in-editor completions, https://github.com/github/copilot-docs/blob/main/docs/jetbrains/gettingstarted.md#3-choosing-alternate-suggestions

What you're seeing is the "Open Copilot" window. Here, you can use TAB to choose the different solutions.
